bodhi/server/tasks/handle_side_and_related_tags.py

```diff
@@ -20,49 +20,56 @@
 import logging
 import typing
 
+import koji  # noqa: 401
+
 from bodhi.server import buildsys, util
 from bodhi.server.models import Update
 
 log = logging.getLogger(__name__)
 
 
-def main(updates: typing.List[dict], from_tag: str):
-    """Handle side-tags and related tags for updates in Koji.
-
-    Args:
-        updates: one or more Update objects
-        from_tag: the tag into which the builds were built
-    """
-    koji = buildsys.get_session()
+def main(aliases: typing.List[str], from_tag: str):
+    """Handle side-tags and related tags for updates in Koji."""
```

Comment:
We lost a quite a bit of the doc string here, is that intended?

Reply:
> We lost a quite a bit of the doc string here, is that intended?

Yes we are now typing the arguments there is no need to duplicate that in the doc string. In my opinion it is better to have that info in only one place.

Reply:
ha just realize the type of args was not in the orginal docstring. I guess we can keep it them :)